app/models/cloud_subnet.rb

```diff
@@ -55,6 +55,25 @@ def self.class_by_ems(ext_management_system)
     ext_management_system && ext_management_system.class::CloudSubnet
   end
 
+  def delete_cloud_subnet_queue(userid)
+    task_opts = {
+      :action => "deleting Cloud Subnet for user #{userid}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'raw_delete_cloud_subnet',
+      :instance_id => id,
+      :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
The openstack method shouldn't have had this at HIGH_PRIORITY, you can drop the `:priority` arg completely from here since it defaults to normal priority.